### DISCUSSION TITLE
Welcome to tvhplayer Discussions!

### DISCUSSION THREAD

**Discussion by mfat** [mfat/tvhplayer#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LouisBTX** [comment]
Virus Total indicating malware in TVHplayer3.5-Setup.exe, is this any concern or false positives?  Thanks!

**Comment by mfat** [comment]
@LouisBTX Hey thanks for reporting this.  It's definitely a false positive.

**Reply by LouisBTX** [reply]
Thanks

**Comment by imans314** [comment]
is it possible to play recordings?
